docs/test-suite/ReFrame-configuration-file.md

```diff
@@ -328,9 +328,9 @@ The most common configuration items defined at this level are:
   [`eessi.testsuite.constants.SCALES`](https://github.com/EESSI/test-suite/blob/main/eessi/testsuite/constants.py)
   and define at which scales tests should be run, e.g. single core, half a node, a full node, two nodes, etc. This
   can be used to exclude running at certain scales on systems that would not support it. E.g. some systems might not
-  support requesting multiple partial nodes, which is what the `1_cpn_2_nodes` (1 core per node, on two nodes) and
-  `1_cpn_4_nodes` scales do. One could exclude these by setting e.g.
-  `features: [FEATURES[CPU]] + [s for s in SCALES if s not in ['1_cpn_2_nodes', '1_cpn_4_nodes']]`. With this
+  support requesting multiple partial nodes, which is what the `1cpn_2_nodes` (1 core per node, on two nodes) and
```

Comment:
It's actually `1cpn_2nodes`, you'll need to remove one more underscore :)

Reply:
Done!